xcp_d/utils/confounds.py

```diff
@@ -351,9 +374,9 @@ def load_confound_matrix(
     elif params == "36P":  # Get rot and trans values, as well as derivatives, WM, CSF,
         # global signal, and square. Add the square and derivative of the WM, CSF
         # and global signal as well.
-        rot_values = confoundtsv[["rot_x", "rot_y", "rot_z"]]
         trans_values = confoundtsv[["trans_x", "trans_y", "trans_z"]]
-        motion = pd.concat([rot_values, trans_values], axis=1)
+        rot_values = confoundtsv[["rot_x", "rot_y", "rot_z"]]
+        motion = pd.concat([trans_values, rot_values], axis=1)
```

Comment:
```suggestion
        motion = confoundtsv[["trans_x", "trans_y", "trans_z", "rot_x", "rot_y", "rot_z"]]
```

Reply:
Also, can you change `confoundtsv` to `confounds_df` and `confoundjson` to `confounds_metadata`?